README.md

```diff
@@ -588,6 +588,12 @@ var signIn = new OktaSignIn(config);
     helpSupportNumber: '(123) 456-7890'
     ```
 
+- **signOutUrl:** url to redirect when a user clicks the sign out link. If not provided, the widget will navigate to primary auth.
```

Comment:
We have a couple different ways of setting links now, I think it'd be nice to:

1. Call this **signOutLink** - we refer to it below as **signOutLinkInMFA**. Also matches helpLinks.
2. Capitalize url since that's what we're doing elsewhere
3. Create a section "Customizing links and buttons" or something like that and group helpLinks, customButtons, registration, and this there.